packages/document/main-doc/docs/en/configure/app/source/design-system.mdx

```diff
@@ -1156,20 +1156,20 @@ export default defineConfig({
 });
 ```
 
-### Configure reference
+### Configuration Reference
 
-With the exception of `screens`, `colors`, and `spacing`, all properties in the configuration object map to the core plugin of `Tailwind CSS`. Since many plugins are responsible for accepting only CSS properties for static value sets (eg `float`), note that not every plugin has corresponding properties in the theme object.
+In addition to `screens`, `colors`, and `spacing`, all properties in the configuration object map to core plugins in `Tailwind CSS`. Because many plugins are responsible for CSS properties that only accept a set of static values (e.g. `float`), not every plugin has a corresponding property in the theme object.
```

Comment:
```suggestion
In addition to `screens`, `colors`, and `spacing`, all properties in the configuration object map to core plugins in TailwindCSS. Because many plugins are responsible for CSS properties that only accept a set of static values (e.g. `float`), not every plugin has a corresponding property in the theme object.
```